sklearn/feature_selection/_univariate_selection.py

```diff
@@ -288,6 +290,189 @@ def chi2(X, y):
     return _chisquare(observed, expected)
 
 
+def _get_entropy(prob):
+    t = np.log2(prob)
+    t[~np.isfinite(t)] = 0
+    return np.multiply(-prob, t)
+
+
+def _info_gain(X, y, aggregate, ratio=False):
+
+    def _a_log_a_div_b(a, b):
+        with np.errstate(invalid="ignore", divide="ignore"):
+            t = np.log2(a / b)
+        t[~np.isfinite(t)] = 0
+        return np.multiply(a, t)
+
+    f_count, c_count, fc_count, total = _get_fc_counts(X, y)
+
+    f_prob = f_count / f_count.sum()
+    c_prob = c_count / c_count.sum()
+    fc_prob = fc_count / total
+
+    c_f = _a_log_a_div_b(fc_prob, c_prob * f_prob)
```

Comment:
To give an example regarding the base, here it would be equivalent to:

```python
c_f = rel_entr(fc_prob, c_prob * f_prob) / np.log(2)
```

Reply:
Yes, that worked.